### ISSUE TITLE
cust.dtsi missing POCO M3 Pro camellia-r-oss

### ISSUE THREAD

**Issue by suiqa** [MiCode/Xiaomi_Kernel_OpenSource#2351]
DTC     arch/arm64/boot/dts/mediatek/camellia.dtb
../arch/arm64/boot/dts/mediatek/camellia.dts:457:10: fatal error: 'camellia/cust.dtsi' file not found
#include "camellia/cust.dtsi"                                                  ^~~~~~~~~~~~~~~~~~~~

**Comment by willzyx-hub** [comment]
> DTC arch/arm64/boot/dts/mediatek/camellia.dtb ../arch/arm64/boot/dts/mediatek/camellia.dts:457:10: fatal error: 'camellia/cust.dtsi' file not found #include "camellia/cust.dtsi" ^~~~~~~~~~~~~~~~~~~~

Yes that's file is missing from source too... Since this is open source why can't you complete it ? @mi-code

**Comment by MTK0DIMENSITY0GANG** [comment]
@mi-code

**Comment by suiqa** [comment]
@mi-code

**Comment by androiddisk** [comment]
@mi-code

**Comment by AbzRaider** [comment]
you should generate that using dct tools anyways there is missing lcm drivers so no use
